framework_tvb/tvb/core/neocom/h5.py

```diff
@@ -134,7 +137,7 @@ def load_from_dir(base_dir, gid, recursive=False):
     return loader.load(gid)
 
 
-def store_to_dir(base_dir, datatype, recursive=False):
+def store_to_dir(datatype, base_dir, recursive=False):
```

Comment:
If you change such signatures, than you should also change the `type` bellow, and the unit-test